src/App.js

```diff
@@ -8,43 +9,102 @@ import CatShow from "./pages/CatShow"
 import Home from "./pages/Home"
 import NotFound from "./pages/NotFound"
 import "./App.css"
-import { Routes, Route } from "react-router-dom"
-import mockCats from "./mockCats"
 
 const App = () => {
-  const [cats, setCats] = useState(mockCats)
-  const createNewCat = (newCat) => {
-    console.log(newCat)
+  const [cats, setCats] = useState([])
+  useEffect(() => {
+    readCat()
+  }, [])
+  const readCat = () => {
+    fetch("http://localhost:3000/cats")
+      .then((response) => response.json())
+      .then((payload) => setCats(payload))
+      .catch((error) => console.log(error))
+  }
+  const getCats = async () => {
+    try {
+      const getResponse = await fetch("http://localhost:3000/cats")
+      if (!getResponse.ok) {
+        throw new Error("Error on the get request for cats")
+      }
+      const getResult = await getResponse.json()
+      setCats(getResult)
+    } catch (error) {
+      alert("Ooops something went wrong:", error.message)
+    }
+  }
```

Comment:
It looks like the readCat function and the getCats function are doing the exact same thing. You can just pick one and disregard the other.